api/db.py

```diff
@@ -178,20 +178,37 @@ def fetchNewWorks(self, page=0, size=50):
         return (baseQuery.count(), baseQuery.offset(offset).limit(size).all())
 
     def fetchSingleCollection(self, uuid):
-        return self.session.query(Collection)\
-            .options(joinedload(Collection.editions))\
-            .filter(Collection.uuid == uuid).one()
+        return (
+            self.session.query(Collection)
+                .options(
+                    joinedload(Collection.editions),
+                    joinedload(Collection.editions, Edition.links),
+                    joinedload(Collection.editions, Edition.items),
+                    joinedload(Collection.editions, Edition.items, Item.links),
+                    joinedload(Collection.editions, Edition.items, Item.rights),
+                )
+                .filter(Collection.uuid == uuid).one()
+        )
```

Comment:
Good change in making the query load all the database models at once to avoid repeated queries to different models.